Gopkg.toml

```diff
@@ -61,6 +61,10 @@
   name = "github.com/spf13/cobra"
   version = "0.0.3"
 
+[[constraint]]
+  name = "k8s.io/kube-state-metrics"
+  version = "v1.6.0-rc.0"
```

Comment:
TODO(me): Need to replace once the release is cut, it should be this week, they already started the process.